### ISSUE TITLE
[designspaceLib] Add lib key for name table name ID 25 (Variations PostScript Name Prefix)

### ISSUE THREAD

**Issue by madig** [fonttools/fonttools#1741]
Variable fonts may need a name ID 25 (e.g. to avoid this bug in Photoshop CC 2019: https://feedback.photoshop.com/photoshop_family/topics/photoshop-cc-2019-variable-fonts-with-separate-fonts-for-uprights-and-italics-snap-back-to-upright-when-changing-an-axis). See https://docs.microsoft.com/en-us/typography/opentype/spec/name#name-ids.

It can be put into the default font's `openTypeNameRecords`, but if you have multiple Designspaces referencing different subsets of the same set of UFOs, they might need different name ID 25s. Instances also don't need it. The field should therefore be settable in the Designspace, in something like a `public.openTypeVariationsPostScriptNamePrefix` lib key.

**Comment by anthrotype** [comment]
Yes, I think we should define a `public.openTypeVariationsPostScriptNamePrefix` DS lib key for that. Ufo2ft should set it accordingly after having generated the VF.

**Comment by madig** [comment]
@josh-hadley told me that Adobe apps expect a unique ID 25 per distinct (VF) file. So the output filename stem (filtered according to spec) could be the default value. Or the DSv5 VF name.

**Comment by Eigi** [comment]
There is a similar bug in InDesign 18.1 (Windows) as described in the initial comment. It happens only if the fonts are installed in the Windows system Fonts-Folder. It does not happen if the fonts are installed in the InDesign fonts folder.
Please define a `public.openTypeVariationsPostScriptNamePrefix` DS lib key for that.
